packages/ui/dev/scripts/generators/generateWorkingGroups.js

```diff
@@ -18,6 +18,8 @@ const generateWorkingGroups = () => {
       description: randomMarkdown(),
       setAtBlockId: randomFromRange(1, 50),
       setAtTime: '2021-03-09T10:28:04.155Z',
+      status: 'Scheduled downtime',
```

Comment:
Let's have that randomized - not every group need to be in downtime :D

I'd go with either:

*   Math.random() > xxx
*   or `id % 3  === 0` to have every 3rd one have a status.

Reply:
Additionally, you can use `faker.lorem.words( randomFromRange(2, 3))`